html/index.html.haml

```diff
@@ -455,15 +88,33 @@
 
 
       #content_container{"ng-controller" => "showController"}
-        %div{"src" => "'scope_os'", "ng-include" => true}
-        %div{"src" => "'scope_packages'", "ng-include" => true}
-        %div{"src" => "'scope_patterns'", "ng-include" => true}
-        %div{"src" => "'scope_repositories'", "ng-include" => true}
-        %div{"src" => "'scope_users'", "ng-include" => true}
-        %div{"src" => "'scope_groups'", "ng-include" => true}
-        %div{"src" => "'scope_services'", "ng-include" => true}
-        %div{"src" => "'scope_config_files'", "ng-include" => true}
-        %div{"src" => "'scope_changed_managed_files'", "ng-include" => true}
-        %div{"src" => "'scope_unmanaged_files'", "ng-include" => true}
 
-        %div{"src" => "'inspection_details_template'", "ng-include" => true}
+        = render_scope "os"
+        = render_scope "packages"
+        = render_scope "patterns"
+        = render_scope "repositories"
+        = render_scope "users"
+        = render_scope "groups"
+        = render_scope "services"
+        = render_scope "config_files"
+        = render_scope "changed_managed_files"
+        = render_scope "unmanaged_files"
+
+        #inspection_details.hidden
```

Comment:
Classes should be listed before IDs (.hidden should precede #inspection_details)